zebra-rpc/src/methods/tests/snapshots/getrawtransaction_invalid_txid@mainnet_10.snap

```diff
@@ -0,0 +1,11 @@
+---
+source: zebra-rpc/src/methods/tests/snapshot.rs
+expression: rsp
+snapshot_kind: text
```

Comment:
Unsure if we should/can just remove this line from the snapshots.

Reply:
You mean the `snapshot_kind`? What would be the motivation?

Reply:
Only the snapshots of this PR has the `snapshot_kind`.

Reply:
I think it's fine as it is.